compass/list.py

```diff
@@ -97,7 +99,13 @@ def list_suites(cores=None):
         for suite in sorted(suites):
             if suite.endswith('.txt'):
                 print('  -c {} -t {}'.format(core, os.path.splitext(suite)[0]))
-
+                if verbose:
+                   text = resources.read_text('compass.{}.suites'.format(core), suite)
+                   tests = list()
+                   for test in text.split('\n'):
+                      test = test.strip()
+                      if len(test) > 0 and test not in tests and not test.startswith('#'):
+                         print("\t* {}".format(test))
```

Comment:
```suggestion
                      if len(test) > 0 and test not in tests and \
                              not test.startswith('#'):
                          print("\t* {}".format(test))
```

Reply:
One more thing. I'm not sure why pep8speaks didn't point it out but there need to be 2 blank lines between functions (at line 109).